tests/api/test_circ_policies_rest.py

```diff
@@ -57,7 +57,7 @@ def test_circ_policies_permissions(
     )
 
     res = client.delete(item_url)
-    assert res.status_code == 403
+    assert res.status_code == 401
```

Comment:
to check

Reply:
401 Unauthorized cause we do not have a logger user here.